xml/issue3753.xml

```diff
@@ -239,6 +238,96 @@ entities.</del>
 </li>
 </ol>
 
+</blockquote>
+
+<note>2022-11-09; Ben Craig and Tomasz provides improved wording</note>
+<p>
+This new resolution merges definition of freestanding item for entity in macro int bullet lists.
+It still n addresses both 3754 and LWG <iref ref="3815"/>.
+</p>
+
+</discussion>
+
+<resolution>
+
+<p>
+This wording is relative to <paper num="N4917"/>.
+</p>
+
+<ol>
+<li><p>Modify 16.3.3.6 <a href="https://wg21.link/freestanding.entity">[freestanding.entity]</a> as indicated:</p>
+
+<blockquote class="note">
+<p>
+[<i>Drafting note:</i> Replace the section name [freestanding.entity] by [freestanding.item] throughout the working draft]
+</p>
+</blockquote>
+
+<blockquote>
+<p>
+<b>16.3.3.6 Freestanding <del>entities</del><ins>items</ins> [freestanding.<ins>item</ins><del>entity</del>]</b>
+<p/>
+-1- A <i>freestanding <del>entity</del><ins>item</ins></i> is <del>a declaration</del><ins>an entity</ins> or
+macro definition that is present in a freestanding implementation and a hosted implementation.
+<p/>
+-2- Unless otherwise specified, the requirements on <ins>non-namespace </ins>freestanding <del>entities</del><ins>items</ins> on a freestanding implementation are the same as the corresponding requirements in a hosted implementation.
```

Comment:
-2- Unless otherwise specified, the requirements on freestanding <del>entities</del><ins>items, except namespaces,</ins> on a freestanding implementation are the same as the corresponding requirements in a hosted implementation.

Reply:
Should be fixed.